autogpt_platform/frontend/src/components/feature-flag/with-feature-flag.tsx

```diff
@@ -0,0 +1,46 @@
+"use client";
```

Comment:
This component needs React hooks e.g [use effect](https://github.com/Significant-Gravitas/AutoGPT/pull/8847/files/a42e563aa662b8be17dc535cc05ff76c3f816a42#diff-33a8a6aff2e5117038be99b173a6490aa4a91caa78d8fc98b4177ac5cef48c7aR16).  In particular because we're responding to real time updates from LaunchDarkly (see useFlags). So can't run this server side